src/SDKs/AnalysisServices/AnalysisServices.Tests/Helpers/AnalysisServicesTestUtilities.cs

```diff
@@ -39,6 +40,8 @@ public static class AnalysisServicesTestUtilities
         public static string DefaultResourceGroup = "TestRG";
         public static string DefaultServerName = "azsdktest";
         public static string DefaultLocation = "West US";
+        public static string DefaultGatewayResourceId = "/subscriptions/ba59a556-5034-4bbb-80b4-4c37cf1083e9/resourceGroups/TestRG/providers/Microsoft.Web/connectionGateways/azsdktest";
```

Comment:
@taiwu please use test framework connection string to inject custom test data, do not hard code it.
https://github.com/Azure/azure-powershell/blob/preview/documentation/Using-Azure-TestFramework.md#5-change-test-environment-settings-at-run-time